litmus-portal/frontend/src/redux/reducers/analytics.ts

```diff
@@ -34,8 +36,13 @@ export const communityData = createReducer<CommunityData>(initialState, {
         count: c[3],
       });
     });
+<<<<<<< HEAD
 
+    const { geoCountry } = data.google;
```

Comment:
This is not required

Reply:
removed..

Reply:
It's still there. I think your commits had some problem maybe.

Reply:
yeah..now its removed..

Reply:
These changes have been made please verify @rajdas98 @gdsoumya and @arkajyotiMukherjee .